src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
```

Comment:
Make Random object and colors arr a class field

Reply:
Not fixed. Yoyu don't want to create new Random each time you call getRandomColor(), you can reuse already created Random, which is declared on class level